File: submissions/abs/abc086_a.rs
use proconio::{fastout, input};
#[fastout]
fn main() {
    input! {
        a: i32,
        b: i32,
    }
    if (a * b) % 2 == 1 {
        println!("Odd");
    }
    else {
        println!("Even");
    }
}

File: submissions/abs/abc086_a.ts
import * as fs from "fs";
const inputs = fs.readFileSync("/dev/stdin", "utf8").split(" ");
const a = +inputs[0];
const b = +inputs[1];
if ((a * b) % 2 === 1) {
    console.log("Odd");
}
else {
    console.log("Even");
}